software/contrib/egressus_melodiam.md

```diff
@@ -0,0 +1,101 @@
+output_2: LFO / randomly generated CV pattern (0 to +5V)
+output_3: LFO / randomly generated CV pattern (0 to +5V)
+output_4: LFO / randomly generated CV pattern (0 to +5V)
+output_5: LFO / randomly generated CV pattern (0 to +5V)
+output_6: LFO / randomly generated CV pattern (0 to +5V)
+
+# Getting started
+
+1. Patch a 50% duty cycle square wave or clock with pulses >= 18ms in duration  into the digital input
+2. Connect one or more outputs to an input on another module (e.g. CV modulation inputs)
+3. Select a pattern pength using Knob 1
+4. Start your clock. Each output will now send a random looping CV to your module!
+
+So, what happened in the above example?
+When the module first powered on it automatically generated 6 x 32 step random CV patterns - one for each of the 6 outputs.
+Each time a clock is received, the step advances by one step and then loops when it get to the end of the pattern.
+The length of the pattern loop is controlled using knob 1, which supports a value from 1 to 32.
+
+# Changing the wave shape and clock division of an output
+
+1. Press and release button 2 until the output number you would like edit is shown on the top-right.
+2. Press button 1 to cycle through the available wave shapes
+3. Adjust knob 2 to select the output division. An output division of 1 causes the CV pattern / LFO for that
+output to run at the clock rate. An output greater than one reduces the CV pattern / LFO to run at the selected division (e.g. selecting a division of 2 would run at half the clock rate)
+
+In LFO mode (pattern length of 1) the wave shape determines the shape of the cycling LFO. However, when in CV pattern mode (Pattern length > 1) the wave shape determines the slew between pattern steps.
+
+# Generating a new CV pattern for an output
+
+A new CV pattern is generated for the selected output by holding down button 1 for 2 seconds and releasing. An indicator is shown on the top left of the screen to show a new CV pattern has been generated. Note that if you are in LFO mode (pattern length of 1) this function will have no effect until the pattern length is increased. 
+
+# LFO Mode / CV Pattern mode
+
+Selecting a pattern length of 1 will output an LFO.
+Selecting a pattern length greater than one plays through the generated CV patterns.
+Slew is generated between CV pattern steps when a wave shape other then square is selected.
+
+# Clocked / Free-running mode
+
+Clocked mode is selected by default - indicated by showing the length of the CV pattern (in dots) in the middle of the screen.
+To enter free-running mode, hold button 2 for 2 seconds and release. The configured clock rate in milliseconds is shown
+in the middle of the screen to indicate you are in free-running mode.
+
+Note that when in free-running mode, the previously selected pattern length remains unchanged, it is therefore a good idea to select the required pattern length before changing to unclocked mode.
+
+# Display
+

```

Comment:
[optional]
A PNG mockup or photo of the GUI with each of the items listed below with numbers could be helpful. There's a lot going on in the GUI, and having a labelled reference image could be useful if you've got the time to put it together.  Then instead of "Left/Middle/Right" you can have the numbers of each item.

Reply:
done, good suggestion. I used this approach one the CVecorder which seemed to work quite well.